sensor_msgs/PointCloud2 cloud
---
sensor_msgs/PointCloud2 segmented_cloud

